package.json

```diff
@@ -3,16 +3,18 @@
   "version": "1.0.0-beta.1",
   "description": "",
   "dependencies": {
-    "@1hive/apps-token-manager": "2.1.0",
+    "@1hive/apps-token-manager": "2.3.0",
```

Comment:
I think this should be `2.2.0` version ? Thats the latest i see published

Reply:
Ah yeah, had to link it to a local version and was confused about how version numbers worked. Will update.